store-conf/conf/imap.log4j.properties

```diff
@@ -0,0 +1,53 @@
+# An example log4j configuration file that outputs to System.out.  The
```

Comment:
Remove this part of the comment - I assume it isn't true...

Have you considered whether we need an analog to `log4j.properties.in`?